module_intros/pmft.PMFTXY.ipynb

```diff
@@ -0,0 +1,252 @@
+{
```

Comment:
<p>Can this be simplified using <code>pmft.plot</code>?</p><p> </p><p>Also, <code>system</code> kwarg.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/freud-examples/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>